test/buildifier.test.ts

```diff
@@ -29,32 +29,55 @@ const workspacePath = path.join(
 
 describe("buildifier", () => {
   it("diagnostics are added from buildifier", async () => {
+
+    // Promise that resolves when diagnostics are added to the file.
+    const diagnosticsPromise = new Promise<vscode.Diagnostic[]>(
+      (resolve, reject) => {
+        const timeout = setTimeout(() => {
+          disposable.dispose();
+          reject(new Error("Timed out waiting for diagnostics"));
+        }, 5000);
```

Comment:
I guess this timeout could also be removed since the tests have their own timeout. What do you think?

Reply:
Yeah I think relying on the test timeout is fine here, and would simplify the test a bit.